libyui-qt/src/YQDumbTab.cc

```diff
@@ -175,18 +240,18 @@ YQDumbTab::preferredHeight()
     int tabBarHeight = _tabBar->sizeHint().height();
     int childHeight  = hasChildren() ? firstChild()->preferredHeight() : 0;
 
-    return tabBarHeight + YQDumbTabSpacing + childHeight;
+    return tabBarHeight + childHeight;
 }
 
 
 void
 YQDumbTab::setSize( int newWidth, int newHeight )
 {
-    QWidget::resize( newWidth, newHeight );
+    // yuiDebug() << "new size: " << QSize( newWidth, newHeight ) << endl;
```

Comment:
NP: I'd either uncomment it (it's a debug only so it won't pollute the log in most cases) or just delete the line, I do not like dead code... :wink:

Reply:
No, I intentionally left that to be easily reactivated when needed, yet not clutter the y2log all the time.